src/pages/App.tsx

```diff
@@ -248,22 +244,8 @@ function UserPropertyUpdater() {
   }, [isDarkMode])
 
   useEffect(() => {
-    if (isUniswapXDefaultLoading || !rehydrated) return
-
-    // If we're not in the transition period to UniswapX opt-out, set the router preference to whatever is specified.
-    if (!isUniswapXDefaultEnabled) {
-      user.set(CustomUserProperties.ROUTER_PREFERENCE, routerPreference)
-      return
-    }
-
-    // In the transition period, override the stored API preference to UniswapX if the user hasn't opted out.
```

Comment:
for the amplitude user property, we can now use the redux value directly every time